Dockerfile.clang

```diff
@@ -20,6 +20,7 @@ COPY test/hello.c ./
 
 RUN \
     for arch in x86 x64 arm arm64; do \
+        ([ $arch = arm ] && printf "%s\n" 10.0.26100.0 $(bash -c ". /opt/msvc/bin/$arch/msvcenv.sh && echo $SDKVER") | sort -VC) || \
```

Comment:
These invocations here need escaping the `$` in `$SDKVER`, otherwise `$SDKVER` gets expanded before invoking `bash`.

Overall - this expression for comparing version numbers does work, but is quite complex (including the nuances for getting the variable escaping correct); I wonder if it'd be simpler to do something like `[ -d /opt/msvc/kits/10/lib/*/um/arm ]` instead?

Reply:
Yes. For this helloworld exmaple, a simpler check is enough, I changed to
```shell
[ ! -d /opt/msvc/kits/10/lib/*/um/$arch ]
```

But I would like to retain the current form in `test.sh`, which explicitly excludes unsupported tests to help identify errors, such as failures in the installation script.

Reply:
Thanks, that sounds reasonable!